<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddContact</class>
 <widget class="QWidget" name="AddContact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddContact</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 25px;
</string>
     </property>
     <property name="text">
      <string>Add Contact</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 0;</string>
     </property>
     <property name="text">
      <string>Phone Number</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="phoneIn">
     <property name="styleSheet">
      <string notr="true">background-color: &quot;#E1ECE1&quot;;
padding: 8px 5px;
border-radius: 10px;
font-size: 14px;</string>
     </property>
     <property name="placeholderText">
      <string>Phone Number</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="phoneErr">
     <property name="styleSheet">
      <string notr="true">color: &quot;red&quot;;\n
padding: 0;\n
padding-left: 5px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="addBtn">
        <property name="styleSheet">
         <string notr="true">border-radius: 10px; 
background-color: &quot;#D3E8D2&quot;;
padding: 10px;
font-weight: 600;</string>
        </property>
        <property name="text">
         <string>Add Contact</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
